python/cuml/test/dask/test_nearest_neighbors.py

```diff
@@ -59,6 +59,12 @@ def _prep_training_data(c, X_train, partitions_per_worker):
     return X_train_df
 
 
+def _scale_rows(client, nrows):
```

Comment:
This reminds me- I've been meaning to check in some changes to make client a fixture instead of cluster.